### PR TITLE
Refactors multiclass perceptron to handle multiple classes

### PR DESCRIPTION
**Pull request by abhi12ravi**

### CODE REVIEW

**Comment by abhi12ravi** [comment]
@drdraze @ThePandaJam Kindly review and merge.

File: discriminant-functions.py [hunk 1]
```diff
@@ -1,42 +1,60 @@
...
   # augmented_matrix is the augmented feature vector from the question
   # eta is the learning rate given in the question
   # omega is an array containing all the output classes of the feature vectors
 
   N_counter = 0 # counter to check for convergence
 
-  # Step 2. Initialize at wieght to 0
-  at = np.array([[0,0,0],[0,0,0],[0,0,0]])
+  #Step 2. Initialise aj for each class
+  at = np.zeros((number_of_classes, number_of_classes))
```
Comment (drdraze) [inline discriminant-functions.py]: should this be num classes x numfeatures?

**Comment by abhi12ravi** [comment]
@drdraze I've added number_of_features. Is it fine now?

**Comment by abhi12ravi** [comment]
@drdraze Let me know if there are any corrections, I've merged it for now.

**Pull request merged**